src/PowerGridComponent.php

```diff
@@ -228,6 +228,11 @@ public function fillData()
 
             $results = $this->applySorting($filters);
 
+            if ($this->headerTotalColumn || $this->footerTotalColumn) {
+                $this->withoutPaginatedData = $results->values()
+                    ->map(fn ($item) => (array) $item);
```

Comment:
Wouldn't it be interesting to go through the transform method here? See this issue in this issue

https://github.com/Power-Components/livewire-powergrid/pull/222

Reply:
I don't think so.
It transformation using only in group functions. 
view using array for showing aggregates values, but transform method return objects inside that using in other places